blocks/results-list-block/features/results-list/results/index.jsx

```diff
@@ -0,0 +1,347 @@
+  const url = websites[arcSite].website_url;
+  return (
+    <div className="list-item" ref={ref}>
+      {(promoElements.showImage)
+        ? (
+          <div className="results-list--image-container mobile-order-2 mobile-image">
+            <a
+              href={url}
+              title={headlineText}
+              aria-hidden="true"
+              tabIndex="-1"
+            >
+              <Image
+                {...imageProperties}
+                url={imageURL !== null ? imageURL : targetFallbackImage}
+                alt={imageURL !== null ? headlineText : imageProperties.primaryLogoAlt}
+                resizedImageOptions={imageURL !== null
+                  ? extractResizedParams(element)
+                  : placeholderResizedImageOptions}
+              />
+            </a>
+          </div>
+        )
+        : null}
+      {(promoElements.showHeadline)
+        ? (
+          <div className="results-list--headline-container mobile-order-1">
+            <a href={url} title={headlineText}>
+              <Heading className="headline-text">{headlineText}</Heading>
+            </a>
+          </div>
+        )
+        : null }
+      {(promoElements.showDescription || promoElements.showDate || promoElements.showByline)
+        ? (
+          <div className="results-list--description-author-container mobile-order-3">
+            {(promoElements.showDescription && descriptionText)
+              ? (
+                <a href={url} title={headlineText}>
+                  <SecondaryFont as="p" className="description-text">
+                    {descriptionText}
+                  </SecondaryFont>
+                </a>
+              )
+              : null}
+            {(promoElements.showDate || promoElements.showByline)
+              ? (
+                <div className="results-list--author-date">
+                  {(promoElements.showByline)
+                    ? (
+                      <Byline
+                        content={element}
+                        list
+                        separator={promoElements.showDate}
+                        font="Primary"
+                      />
+                    )
+                    : null }
+                  {(promoElements.showDate)
+                    ? <ArticleDate classNames="story-date" date={displayDate} />
+                    : null }
+                </div>
+              )
+              : null}
+          </div>
+        )
+        : null }
+    </div>
+  );
+}));
+
+const Results = ({
+  customFields,
+  fusionContext,
+  fusionProperties,
+  imageProperties,
+}) => {
+  const {
+    lazyLoad,
+    listContentConfig: {
+      contentService,
+      contentConfigValues,
+    },
+  } = customFields;
+
+  const {
+    arcSite,
+    contextPath,
+    deployment,
+    isAdmin,
+  } = fusionContext;
+
+  const {
+    fallbackImage,
+    locale,
+  } = fusionProperties;
+
+  const [configuredOffset] = useState(
```

Comment:
What's the reason to use state to store everything in?

Reply:
I explained more in the comment to Jack below, but it's more to prevent re-renders. I don't need them on the simple values and am checking in code to revert some of that useState.

Reply:
I'm not able to replicate the reason for dismissing this. I was able to replicate this suggestion helping https://github.com/WPMedia/fusion-news-theme-blocks/pull/967/files

Reply:
I assume resolved since you close the branch?